src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/EntityViewerSidebarModal.tsx

```diff
@@ -36,6 +36,9 @@ const entityViewerSidebarModals: Record<
   [SidebarModalView.BOOKMARKS]: lazy(
     () => import('./modal-views/EntityViewerBookmarks')
   ),
+  [SidebarModalView.SHARE]: lazy(
+    () => import('./modal-views/EntityViewerShare')
+  ),
```

Comment:
I'm wondering whether we shouldn't just delete `./modal-views/EntityViewerShare` and disable the share button in the toolbar. We aren't going to have a share functionality for the MAP, and maybe not even for the MVP; so we shouldn't really keep those empty components.